rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -70,10 +72,21 @@ extern "C"  RocalTensor  ROCAL_API_CALL rocalResize(RocalContext context, RocalT
 /// using it directly. This option allows certain optimizations to be achieved.
 /// \param p_mirror Parameter to enable horizontal flip for output image.
 /// \return
-extern "C"  RocalTensor  ROCAL_API_CALL rocalResizeMirrorNormalize(RocalContext p_context, RocalTensor p_input,
-                                                                  unsigned dest_width, unsigned dest_height,
-                                                                  std::vector<float> &mean, std::vector<float> &std_dev,
-                                                                  bool is_output, RocalIntParam p_mirror = NULL);
+extern "C" RocalTensor ROCAL_API_CALL rocalResizeMirrorNormalize(RocalContext p_context, 
```

Comment:
extern "C"  RocalTensor  ROCAL_API_CALL 

Please maintain the 2 spacing between the "C" and RocalTensor / change it to single space everywhere in the entire file